src/components/Logo.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled, { keyframes } from 'styled-components'
+import HGroup from './HGroup'
+import VGroup from './VGroup'
+import logoImg from '../img/logo.png'
+
+const LogoWrapper = styled(HGroup)`
+  align-items: center;
+  @media (max-width: 960px) {
+    flex-direction: column;
+  }
+`
+
+const TitleWrapper = styled(VGroup)`
+  @media (max-width: 960px) {
+    align-items: center;
+  }
+`
+const spin = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+`
+
+const Image = styled.img`
+  width: 200px;
+  height: auto;
+  margin-bottom: 30px;
+  // animation: ${spin} 10s infinite linear;
```

Comment:
if not used you can remove now

Reply:
forgot to ask, what do you guys think about this? Logo should be spinning or no?
My vote: no

Reply:
not anymore!